lib/svtplay_dl/service/urplay.py

```diff
@@ -72,3 +75,20 @@ def find_all_episodes(self, config):
                 episodes_new.append(i)
             n += 1
         return episodes_new
+
+    def outputfilename(self, data, urldata):
+        if "seriesTitle" in data:
+            self.output["title"] = data["seriesTitle"]
+        if "episodeNumber" in data and data["episodeNumber"]:
+            self.output["episode"] = "{:02d}".format(data["episodeNumber"])
```

Comment:
you can dont have to do the format thing here. it will be done in a later step automagic  :)

Reply:
ok, I looked at the svtplay.py service which also does padding itself?

Reply:
e.g. Line 302 in svtplay.py: `episode = "{:02d}".format(int(match.group(2)))`

Reply:
it might be old  stuff.
we do the zero padding here https://github.com/spaam/svtplay-dl/blob/master/lib/svtplay_dl/utils/output.py#L161  :)